apiserver/common/cloudspec/statehelpers.go

```diff
@@ -27,7 +27,12 @@ func MakeCloudSpecGetter(pool Pool) func(names.ModelTag) (environs.CloudSpec, er
 			return environs.CloudSpec{}, errors.Trace(err)
 		}
 		defer release()
-		return stateenvirons.EnvironConfigGetter{st}.CloudSpec()
+
+		m, err := st.Model()
+		if err != nil {
+			return environs.CloudSpec{}, errors.Trace(err)
+		}
+		return stateenvirons.EnvironConfigGetter{st, m}.CloudSpec()
```

Comment:
Please add a TODO here and in EncironConfigGetter itself - as soon as a few Cloud() methods are moved off State and onto Model, we won't need to pass in both state and model. In fact, we should just continue to just pass in state only. I'm not sure why we need to pass in model here. The CloudSpec() method on EnvironConfigGetter can just continue to call st.Model() itself as it does now. We can change it to use a model directly later when the cloud methods are moved off state